src/lib/stdgunw/defs.h

```diff
@@ -0,0 +1,18 @@
+//
+//  defs.h
+//  GunwOS
+//
+//  Created by Artur Danielewski on 15.01.2023.
+//
+
+#ifndef STDGUNW_DEFS_H
+#define STDGUNW_DEFS_H
+
+#define NULL 0
+#define nullptr 0
+
+#define KB(C) (C * 1024)
+#define MB(C) (KB(C) * 1024)
+#define GB(C) (MB(C) * 1024)
```

Comment:
Żeby być mega dokładnym to powinno być `KiB`, `MiB` i `GiB`: https://startdebugging.net/2020/08/what-is-the-difference-between-a-megabyte-mb-and-a-mebibyte-mib/

Reply:
Słusznie - zmienione